packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -129,10 +134,27 @@ export class ProjectService {
         });
         this.projectLoading[projectUuid] = false;
         this.projectAdapters[projectUuid] = adapter;
+        this.projectModel.addJobStep(jobUuid, 'saving explorers to cache');
+
         this.projectModel.saveExploresToCache(projectUuid, explores);
+        this.projectModel.addJobStep(jobUuid, 'done');
+
         return this.getProject(projectUuid, user);
     }
 
+    async create(user: SessionUser, data: CreateProject): Promise<string> {
+        if (user.ability.cannot('create', 'Project')) {
+            throw new ForbiddenError();
+        }
+
+        const jobUuid = await this.startJob(undefined);
+
+        createProject(user, data);
```

Comment:
called without await, this should make create mostly async, just returning the jobuuid